package.json

```diff
@@ -18,7 +18,7 @@
     "lint:hbs:fix": "ember-template-lint . --fix",
     "lint:js": "eslint . --cache",
     "lint:js:fix": "eslint . --fix",
-    "start": "cross-env IRENE_API_HOST=http://localhost:8000 ember server",
+    "start": "cross-env IRENE_API_HOST=http://localhost:8000 IRENE_API_SOCKET_PATH=http://localhost:8008 ember server",
```

Comment:
`IRENE_API_SOCKET_PATH` is not used anymore...its not required

Reply:
Done 👍